### ISSUE TITLE
url tag doesn't handle https

### ISSUE THREAD

**Issue by paxter** [jbowens/jBBCode#65]
The default url tag defintion ignores the https protocoll and set all urls to http. Escpecially for intern links on a website it's a problem, because all bbcode formatted texts link to http content and ignores if you are browsing the site via https.
Has anybody a quick solution for this problem?

**Comment by DaSourcerer** [comment]
This seems to be a deficiency of `filter_var()`. Implement your own `UrlValidator` and overwrite the default definition for the url-tag.

**Comment by paxter** [comment]
filter_var() doesn't make a difference between https and http.

**Comment by DaSourcerer** [comment]
Yes, that's pretty much what I wrote. Implementing your own variation of `UrlValidator` will allow you to get around that.

**Comment by BenFenner** [comment]
This issue is likely out of date. The URL tag certainly respects the https protocol (scheme) on my site using JBBCode v1.3.0.
It could be that the PHP version bumps from 5 to 7 to 8 solved this issue somewhere along the way as the `filter_var()` behavior using the `FILTER_VALIDATE_URL` tag has certainly changed throughout those versions.
